_topic_maps/_topic_map_ms.yml

```
@@ -195,7 +195,7 @@ Distros: microshift
 Topics:
 - Name: Using Kustomize to deploy applications
   File: microshift-applications
-- Name: Deleting resource manifests
+- Name: Deleting or upgrading Kustomize manifest resources
```

Update is the preferred term per the [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update):
```suggestion
- Name: Deleting or updating Kustomize manifest resources
```

---

done